EightMinuteEmpireLegends/MainDriver/Main.cpp

```diff
@@ -16,9 +16,30 @@
 #include "../BiddingFacility/BidSubmission.h"
 #include "../BiddingFacility/BidTieBreakerByLastName.h"
 
+#include "../Player/Player.h"
+
 int main() {
+
     BidTieBreakerByLastName tieBreaker;
 
+    //----------------------------------------------------------------
+    //                        Player Driver 
+    //----------------------------------------------------------------
+
+    //assuming 2 player game 
+    Deck deck(2);
```

Comment:
Please place the driver code in a separate function called runPlayerDriver(), i will call it in the main since im in charge of the main driver.